packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -190,6 +210,22 @@ class ApplicationCommand extends Base {
     return this.edit({ name });
   }
 
+  /**
+   * Edits the localized names of this ApplicationCommand
+   * @param {Object<string, string>} nameLocalizations The new localized names for the command
+   * @returns {Promise<ApplicationCommand>}
+   * @example
+   * // Edit the name localizations of this command
+   * command.setLocalizedNames({
+   *   'en-US': 'ping',
```

Comment:
wouldn't en-US be the one in `name`? This should probably have a different locale